Microsoft.Toolkit.Uwp/Helpers/Network/ConnectionInformation.cs

```diff
@@ -0,0 +1,139 @@
+﻿// ******************************************************************
+// Copyright (c) Microsoft. All rights reserved.
+// This code is licensed under the MIT License (MIT).
+// THE CODE IS PROVIDED “AS IS”, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED,
+// INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.
+// IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM,
+// DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT,
+// TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH
+// THE CODE OR THE USE OR OTHER DEALINGS IN THE CODE.
+// ******************************************************************
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Windows.Networking.Connectivity;
+
+namespace Microsoft.Toolkit.Uwp
+{
+    /// <summary>
+    /// This class exposes information about the network connectivity.
+    /// </summary>
+    public class ConnectionInformation
+    {
+        private List<string> networkNames = new List<string>();
```

Comment:
Can be readonly? `private List<string> networkNames { get; } = new List<string>();`